base_import_match/models/base_import.py

```diff
@@ -88,7 +89,7 @@ def _compute_name(self):
                 " + ".join(
                     s.field_ids.mapped(
                         lambda r: (
-                            str(r.field_id.name) +
+                            unicode(r.field_id.name) +
```

Comment:
Agreed, method calls in Python are more expensive than boolean evaluations - so the proposed `u'my_field and {}'.format(my_field) or my_field` would be ideal even though more code